doc/relevancy.md

```diff
@@ -0,0 +1,383 @@
+# Relevancy
+
+This document explains how documents are ranked when you perform a search
```

Comment:
I'm not sure of a better way to word it but "this document explains how documents are ranked" feels a bit off, because "document" is used in two different senses in the same clause.

Maybe something like

> This document explains how relevancy ordering works when performing a search